ibis/mapd/api.py

```diff
@@ -38,6 +38,7 @@ def connect(
     port=6274,
     database=None,
     protocol='binary',
+    sessionid=None,
```

Comment:
Can we call this `session_id` to be consistent with other parameters' naming conventions?

Reply:
I had chosen `sessionid` because that is what is used in the underlying `pymapd` library, but if `session_id` is preferable for `ibis`, that works for me.